integration/tests/mysql/Makefile

```diff
@@ -72,29 +78,10 @@ run: 5.6.49 5.7.31 8.0.21
 	make -C auto-increment-create run
 	make -C auto-increment-add run
 	make -C auto-increment-drop run
-
-
-.PHONY: 8.0
-8.0: export MYSQL_VERSION = 8.0
-8.0:
-	make -C decimal run
```

Comment:
why remove all these?

Reply:
this was just adding in a specific patch version, to give better insight into the versions that were tested